src/Uno.UWP/Storage/ApplicationDataContainer.Android.cs

```diff
@@ -21,17 +21,13 @@ public partial class ApplicationDataContainer
 
 		private class SharedPreferencesPropertySet : IPropertySet
 		{
-			private readonly ISharedPreferences _preferences;
-
 			public SharedPreferencesPropertySet()
 			{
-#pragma warning disable 618
-#pragma warning disable CA1422 // Validate platform compatibility
-				_preferences = PreferenceManager.GetDefaultSharedPreferences(ApplicationData.GetAndroidAppContext());
-#pragma warning restore CA1422 // Validate platform compatibility
-#pragma warning restore 618
 			}
 
+			private ISharedPreferences Preferences
```

Comment:
I would turn this into `CreatePreferences` method, so it is clear that a new instance is made each time